commands/upgrade/util.py

```diff
@@ -176,10 +176,14 @@ def prepare_configuration(args):
     if args.channel:
         os.environ['LEAPP_TARGET_PRODUCT_CHANNEL'] = args.channel
 
-    # Check upgrade path and fail early if it's unsupported
-    target_version, flavor = command_utils.vet_upgrade_path(args)
-    os.environ['LEAPP_UPGRADE_PATH_TARGET_RELEASE'] = target_version
-    os.environ['LEAPP_UPGRADE_PATH_FLAVOUR'] = flavor
+    # NOTE(ivasilev) That is a safeguard - if for some reason prepare_configuration is called as part of
+    # leapp upgrade --resume this won't make it crash in target version container for the latest supported
+    # release.
+    if not args.resume:
```

Comment:
Sadly, but as @Rezney has correctly pointed out it has to go - preupgrade doesn't have --resume and this would fail.